### DISCUSSION TITLE
end December data showing in different year in Dashboard

### DISCUSSION THREAD

**Discussion by pmanandhar** [atoti/atoti#800] [General]
I created a date hierarchy and I have my 2021 data  from december 26 to december 31  being shown as belonging to 2022. Is this some kind of bug in the visualization dashboard?

**Comment by Skyscimitar** [comment]
Hey! This is most likely due to using the incorrect date format for your values. If you use `YYYY` to parse the year, then this corresponds to the "week based year", which can result in some dates from the end of December being considered as part of the following year. To fix this you should use `yyyy` to parse the year in your date values.

**Reply by pmanandhar** [reply]
Thank you
